uber/config.py

```diff
@@ -136,6 +136,19 @@ def SUPPORTER_COUNT(self):
                                                 sa.Attendee.amount_paid >= self.SUPPORTER_LEVEL).count()
             return individual_supporters + group_supporters
 
+    @property
+    def SQLALCHEMY_FINAL_URL(self):
+        # support reading the DB connection info from an environment var (used with Docker containers)
+        # DB_PORT_5432_TCP_ADDR="172.17.0.8"
+        # DB_PORT_5432_TCP_PORT="5432"
+        docker_db_addr = os.environ.get('DB_PORT_5432_TCP_ADDR')
+        docker_db_port = os.environ.get('DB_PORT_5432_TCP_PORT')
+
+        if docker_db_addr is not None and docker_db_port is not None:
```

Comment:
Wouldn't we also need to read the db name, username, and password from Docker environment variables as well?  The way this is written, Docket will always use ``m13`` for those fields.